x/group/spec/messages.md

```diff
@@ -0,0 +1,46 @@
+```go
+// Groups get their own GroupID
+type MsgCreateGroup struct {
+	Signer sdk.AccAddress `json:"signer"`
```

Comment:
Why is there a Signer and an Owner?
Either they should be the same, or we don't have to require any Signer (if anyone can create a group)

Reply:
A `Signer` is always required in a cosmos SDK message. One reason for having both is that `Owner` may actually be a contract - for instance some contract that elects the members of the group.

Reply:
I see the purpose of `Owner` in state.
`Signers()` can return `msg.Owner`

If we don't care who signs, no need to return anything.
Again, this is a bit more discussion of best practices of cosmos-sdk... not sure.